src/adapter/services/LegacyObjectAPIInterceptor.js

```diff
@@ -70,7 +70,7 @@ define([
     };
 
     ObjectServiceProvider.prototype.create = async function (object) {
-        let model = utils.toOldFormat(object);
+        let model = await utils.toOldFormat(object);
```

Comment:
`utils.toOldFormat` is not an async call

Reply:
changed to awaiting a variable which then gets returned